### DISCUSSION TITLE
v0.21.3

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1753] [Releases]
## What's Changed
* Validate ping url by @nandi95 in https://github.com/alexjustesen/speedtest-tracker/pull/1662
* Use latest docker tag in README by @earthpyy in https://github.com/alexjustesen/speedtest-tracker/pull/1711
* Bump vite from 5.4.0 to 5.4.6 by @dependabot in https://github.com/alexjustesen/speedtest-tracker/pull/1713
* Bump rollup from 4.20.0 to 4.22.4 by @dependabot in https://github.com/alexjustesen/speedtest-tracker/pull/1724
* Laravel 11.24.1 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1727
* [Bug] Remove writing server_id to an column for failed speedtests by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1679
* Laravel 11.27.2 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1749
* [Chore] improve composer.json by @wilsenhc in https://github.com/alexjustesen/speedtest-tracker/pull/1746
* Release v0.21.3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1752

## New Contributors
* @nandi95 made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/1662
* @earthpyy made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/1711
* @wilsenhc made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/1746

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.21.2...v0.21.3

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.21.3'>v0.21.3</a>.</em>

**Comment by nandi95** [comment]
@alexjustesen Please release https://github.com/alexjustesen/speedtest-tracker/pull/1754 as soon as possible as I imagine all people used ip addresses up to this point which is now breaking

**Reply by alexjustesen** [reply]
`v0.21.4` disables checking for an internet connection which is all I had time for tonight, will fix the regression in the next release.
